tests/integration_tests/analysis/test_es_update.py

```diff
@@ -258,7 +281,7 @@ def test_gen_data_obs_data_mismatch(storage, uniform_parameter):
         name="posterior",
         prior_ensemble=prior,
     )
-    AnalysisConfig()
+    # AnalysisConfig()
```

Comment:
Stray comment, looks like it can just be deleted.

Reply:
Removing